.github/actions/spell-check/expect.txt

```diff
@@ -604,11 +533,9 @@ FILEVERSION
 Filtergraph
 Filterkeyboard
 Filterx
-Finalizers
 findfast
 firefox
 FIXEDFILEINFO
-flyout
 flyouts
```

Comment:
I saw earnings in the last PRs that such entries can get simplyfied to only the singular version. Can you clean these cases up?

Reply:
@jsoref 
Is ist "Singular=Plural" can be simplified or is it "Upper case=Lower case"?

Reply:
What I meant is the following:
![image](https://github.com/microsoft/PowerToys/assets/61519853/e2ef8ca5-ad47-473c-bc7d-e93238de819e)

Reply:
In this case, you can remove `flyouts` as it's covered by `flyout`. I'll have to see how hard it is to get that message to report the more general variant... (The code right now doesn't have that information readily available, but I should be able to do it...)

Reply:
in a diff PR, i removed the plural to be covered by flyout